src/main/java/viscount/Storage.java

```diff
@@ -0,0 +1,125 @@
+package viscount;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import viscount.exception.ViscountSaveDataException;
+import viscount.task.Task;
+
+/**
+ * Represents Viscount's storage.
+ *
+ * Handles loading tasks from data file and writing tasks to the file.
+ */
+public class Storage {
+    private static final String DATA_FILE_NAME = "viscount.txt";
+
+    private String dataDirectoryPath;
+    private String filePathString;
+
+    /**
+     * Instantiates a new storage object.
+     *
+     * @param dataDirectoryPath Directory path to the data file.
+     */
+    public Storage(String dataDirectoryPath) {
+        this.dataDirectoryPath = dataDirectoryPath;
+        this.filePathString = dataDirectoryPath + DATA_FILE_NAME;
+    }
```

Comment:
Do you think it might be a good idea to specify the data directory path inside this class as constant instead of taking it in as an argument because the data directory is fixed anyway?